app/res/values-pt/strings.xml

```diff
@@ -271,5 +271,22 @@
     <string name="preview">Prever</string>
     <string name="show_raw_markdown">Mostrar markdown puro</string>
     <string name="render_markdown">Renderizar markdown</string>
+    <string name="are_you_sure">Você está absolutamente certo?</string>
+    <string name="code_authentication_failed">Falha de autenticação! Código de verificação incorreto</string>
+    <string name="confirm_comment_delete_message">Tem certeza de que deseja excluir este comentário?</string>
+    <string name="confirm_comment_delete_title">Remover Comentário</string>
+    <string name="copy_hash">Copiar hash</string>
+    <string name="delete_cap">EXCLUIR</string>
```

Comment:
Might be better to use be *apagar* or *deletar* in these `delete_*` to better match the English version. Especially in the repository case, *excluir* might read like hiding/omission, which would give the wrong impression given that it is undoable.